app/policies/workflow_execution_policy.rb

```diff
@@ -4,16 +4,20 @@
 class WorkflowExecutionPolicy < ApplicationPolicy
   def export_workflow_execution_data?
     return true if record.submitter.id == user.id
+    return true if Member.can_create_export?(user, record.namespace) == true
 
     details[:id] = record.id
     false
   end
 
   def destroy? # rubocop:disable Metrics/AbcSize
     return true if record.submitter.id == user.id
```

Comment:
**question:** `cancel?` and `destroy?` are identical.  That makes sense to me, is there a way to simplify the redundant code?

Reply:
From Deep: Whenever I get to refactor the policies I'm going to try and make some common functions